.github/workflows/patch-code-of-conduct.yml

```diff
@@ -0,0 +1,26 @@
+name: Patch Code of Conduct
+on: [push]
+
+jobs:
+  main_job:
+    runs-on: ubuntu-latest
+    name: Test Action
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Patch CoC
+        uses: pkgjs/patch-my-code-of-conduct@v1.1.0
+        with:
+          base_url: 'UPDATE_THIS'
```

Comment:
does this need to be updated?

Reply:
Yes, this will be updated when the previous pull request for PREAMBLE is merged. 👍 Not resolving this for future tracking.

Reply:
I updated this with the latest commit URL that changes this. Thanks!